src/java/fr/paris/lutece/portal/web/user/AuthenticationFilter.java

```diff
@@ -75,6 +84,7 @@ public class AuthenticationFilter implements Filter
     private static final String PROPERTY_RESET_EXCEPTION_MESSAGE = "User must reset his password.";
     private static final String PROPERTY_JSP_URL_ADMIN_LOGOUT = "lutece.admin.logout.url";
     private static final String JSP_URL_ADMIN_LOGIN = "jsp/admin/AdminLogin.jsp";
+    private Logger _logger = LogManager.getLogger( "lutece.securityHeader" );
```

Comment:
Please use a static constant

Reply:
Taken into account in commit 93a56b8139c008a7cc3a87cce5a8ccbffb4c9734